### PR TITLE
Rs encoding handler

### PR DESCRIPTION
**Pull request by burdettadam**
This PR contains code for rich schema "encoding" handlers.

### CODE REVIEW

File: indy_node/test/request_handlers/conftest.py [hunk 1]
```diff
@@ -36,7 +42,7 @@ def schema_request():
     return Request(identifier=randomString(),
                    reqId=5,
                    signature="sig",
-                   operation={'type': SCHEMA,
+                   operation={'type': "101",
```
Comment (anikitinDSR) [inline indy_node/test/request_handlers/conftest.py]: Why did you change constant to string?

File: indy_node/test/request_handlers/conftest.py [hunk 2]
```diff
@@ -45,6 +51,32 @@ def schema_request():
                               }})
 
 
+@pytest.fixture(scope="module")
+def rs_encoding_handler(db_manager, write_auth_req_validator):
+    return RsEncodingHandler(db_manager, write_auth_req_validator)
+
+
+@pytest.fixture(scope="function")
+def rs_encoding_request():
```
Comment (anikitinDSR) [inline indy_node/test/request_handlers/conftest.py]: You already have had a function `build_get_rs_encoding_request`. I think it will be more clearly if that function will be used instead of copy-pasting. Also we have a helper function to transform txn to request and request to txn. They are placed in `plenum/common/txn_util.py`

File: indy_node/test/rs_encoding/test_send_rs_get_encoding.py [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        looper, sdk_pool_handle, nodeSet, sdk_wallet_trustee, send_rs_encoding):
+    _, did = sdk_wallet_trustee
+    identifier, type, name, version = send_rs_encoding.split(':')
+    _id = identifier + ':' + type + ':' + name + ':' + "2.0"
+    request = build_get_rs_encoding_request(did, _id)
+    rep = sdk_get_and_check_replies(looper, [sdk_sign_and_submit_req(sdk_pool_handle, sdk_wallet_trustee, request)])
+    assert rep[0][1]['result']['seqNo'] is None
+
+
+def test_send_get_rs_encoding_fails_with_invalid_version_syntax(
```
Comment (anikitinDSR) [inline indy_node/test/rs_encoding/test_send_rs_get_encoding.py]: I think, that this and below tests can be parametrized

Comment (burdettadam) [reply indy_node/test/rs_encoding/test_send_rs_get_encoding.py]: I agree I will add a helper method to cut down on duplicate code.

**Comment by burdettadam** [comment]
closing in favor of indy-2338.
